examples/social-media/readme.md

```diff
@@ -0,0 +1,24 @@
+## Small social media example
+
+Say we have a social network, a very basic one, where all you can do is upvoting comments.
+
+The individual services that need to be defined are:
+
+- A frontend application where users interact through the website, where they can like comments.
```

Comment:
Linking to the documents will be awesome

Reply:
Added 🙂